src/crypto/tls/common.go

```diff
@@ -23,9 +24,21 @@ import (
 // only supports Elliptic Curve based groups. See RFC 8446, Section 4.2.7.
 type CurveID uint16
 
+// CipherSuiteName returns the standard name for the passed cipher suite ID
+//
+// Not Implemented.
+func CipherSuiteName(id uint16) string {
+	return fmt.Sprintf("0x%04X", id)
+}
+
 // ConnectionState records basic TLS details about the connection.
 type ConnectionState struct {
-	// TINYGO: empty; TLS connection offloaded to device
+	// TINYGO: empty; TLS connection offloaded to device [?]
+	//
+	// Minimum (empty) fields for fortio.org/log http logging to
```

Comment:
What is the broader implication? `net/http` compatibility?

Reply:
well... if I ever get a super sample server working with tinygo, it'd have the logging I use elsewhere for servers... I haven't tried others but mine does need these fields even empty to compile

Reply:
in short it's incremental progress - can we merge?